apps/bottle/src/store/query/ServerFetchBoundary.tsx

```diff
@@ -0,0 +1,19 @@
+import { FetchQueryOptions, HydrationBoundary, QueryClient, dehydrate } from '@tanstack/react-query';
```

Comment:
Since `FetchQueryOptions` is a type, it seems better to add 'type' separately! Is there an associated ESLint rule being applied?

```suggestion
import { type FetchQueryOptions, HydrationBoundary, QueryClient, dehydrate } from '@tanstack/react-query';
```

Reply:
What do you think about managing queryClient with useState? It's intended to ensure it's referenced only once in React's lifecycle. I'm curious about your opinion. [document](https://tanstack.com/query/latest/docs/eslint/stable-query-client#rule-details)